src/test/java/com/wikia/webdriver/common/templates/core/CoreTestTemplate.java

```diff
@@ -3,10 +3,13 @@
 import java.io.File;
 import java.lang.reflect.Method;
 
+import net.lightbody.bmp.core.har.Har;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.lightbody.bmp.core.har.Har'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)